src/work-allocation/containers/task-list/task-list.component.html

```diff
@@ -4,11 +4,11 @@
   <!-- Column Headers and Cells -->
   <ng-container *ngFor="let field of fields" [cdkColumnDef]="field.name">
     <th class="govuk-table__header" [attr.aria-sort]="isColumnSorted(field.name)" cdk-header-cell *cdkHeaderCellDef>
-      <button type="button" [id]="'sort_by_' + field.name" data-index="0"
+      <button *ngIf="field.columnLabel" type="button" [id]="'sort_by_' + field.name" data-index="0"
               (click)="onSortHandler(field.name)" >{{field.columnLabel}}</button>
     </th>
     <td class="govuk-table__cell border-none" cdk-cell *cdkCellDef="let rowData">
-      <exui-task-field [task]="rowData" [config]="field"></exui-task-field>
+      <exui-task-field [task]="rowData" [config]="field" [sourceColumn]="field.sourceColumn" [matchValue]="field.matchValue"></exui-task-field>
```

Comment:
The additional parameters are not needed when you're already passing in `field` to the `config` parameter - simply get those values from `config.sourceColumn` and `config.matchValue` within the component.